securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -5,9 +5,9 @@
 
 The library can be retrieved directly from the {project_name} server at `{kc_js_path}/keycloak.js` and is also distributed as a ZIP archive.
 
-A best practice is to load the JavaScript adapter directly from {project_name} Server as it will automatically be updated when you upgrade the server. If you copy the adapter to your web application instead, make sure you upgrade the adapter only after you have upgraded the server.
+A good practice is to include the JavaScript adapter directly into your web application to make sure your web application is self-contained and starts up fast. Keep in mind you need to upgrade the JavaScript adapter after you have upgraded the server. Alternatively, load the JavaScript adapter directly from {project_name} Server. Then it will automatically be updated when you upgrade the server.
 
-NOTE: You can also download package from npm: https://www.npmjs.com/package/keycloak-js
+NOTE: You can also download the package from npm: https://www.npmjs.com/package/keycloak-js
```

Comment:
I would personally start making this the recommended approach, as it allows us to introduce changes to the API without breaking the existing package.